content/en/Getting started/glossary.md

```diff
@@ -75,6 +75,83 @@ refines best practice security solutions.
 One of the test criteria used by our pentesters is [CIS Controls v8](https://www.cisecurity.org/controls/v8),
 released in 2021.
 
+## Cobalt Users
+
+When using the Cobalt UI, you may encounter a variety of different users, in the following
+roles:
+
+- Organization Roles: If you're a Cobalt customer, your account may have one or more of the following roles:
+  - Organization Owner
+  - Organization Member
+  - Pentest Team Member
+
+- Pentester Roles: Cobalt pentesters who are assigned to your pentest have one of two roles:
+  - Lead
+  - Pentester
+
+  Some Cobalt pentesters may be a _Lead_ in one test, a _Pentester_ in a second test, and
+  possibly no role and no involvement in your other pentests.
+
+Select Cobalt employees may be assigned as administrators, as _Cobalt Staff_.
+
+You can review a list of permissions associated with each organization role in the following
+article: [What do the user roles mean?](https://cobaltio.zendesk.com/hc/en-us/articles/360057093472-What-do-the-user-roles-mean-).
+
+<!-- Per https://zombie.atlassian.net/browse/DOCS-5 I should add info from an internal
+spreadsheet, but need to find pull code in Hugo -->
+
+### Organization Owner
+
+An _Organization Owner_ is the administrator for a customer organization within the Cobalt app. As such, they can:
+
+- Add/remove the users of their choice, by their email addresses, as an _Organization Member_
+  or _Organization Owner_.
+- View collaborators, the members of their pentest team. That team includes:
+  - _Pentest Team Member_, typically an organization employee.
+  - _Pentest Lead_, the Cobalt pentester responsible for the pentest.
+  - _Pentester_, one or more additional Cobalt pentesters whose helping with the pentest.
+  Smaller pentests may only have a _Pentest Lead_.
+
+- An _Organization Owner_ may also be a [_Pentest Team Member_](#pentest-team-member).
+
+- If allowed by their {{% ptaas-tier %}}, they can also manage
+[multi-factor authentication (MFA)](#multi-factor-authentication) as well as
+[SAML](#security-assertion-markup-language) settings for the users in their organization.
+
+An _Organization Owner_ also has top-level (sudo) administrative privileges for their
+organizations in the Cobalt app.
+
+### Organization Member
```

Comment:
As requested, I no longer refer to assets in the description for an Organization Member.

Reply:
Actually, changed my mind, based on [@molfinn 's feedback](https://github.com/cobalthq/cobalt-product-public-docs/pull/119#issuecomment-1113388461), specifically:

"An Organization Member is a user of the Cobalt App who can create a [pentest](https://deploy-preview-119--cobalt-docs.netlify.app/getting-started/glossary/#pentest). That user can also see the pentesters who are working on their asset. If allowed by their [PtaaS Tier](https://cobaltio.zendesk.com/hc/en-us/articles/4408839706132-Cobalt-PtaaS-Tiers), they can also manage integration with Jira and GitHub.
• an org member can also create an asset
• is it clear to say that a user can see the pentesters who are working on their asset? pentesters are getting assigned to a pentest. Don't want this to sound like a user can click on assets and see who is assigned to specific assets. Might be overthinking it though." 

Molly, I _think_ the issue is that I didn't really define "Pentest". I've now added lines 247-250, which defines the relationship between a pentest and an asset, as well as the roles played by a "pentester" and a "pentest lead". 

You can preview the appearance and links of my updates in:

- https://deploy-preview-119--cobalt-docs.netlify.app/getting-started/glossary/#organization-member
- https://deploy-preview-119--cobalt-docs.netlify.app/getting-started/glossary/#pentest